ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -894,6 +899,23 @@ mac_address_table:
   aging_time: < aging_time_in_seconds >
 ```
 
+### MACsec
+
+```yaml
+mac-security:
+  license:
+    license_name: < license-name >
+    license_key: < license-number >
+  fips_restrictions: < true | false >
+  profiles:
+  < profile >:
```

Comment:
Profile and following lines missing indention.

Reply:
Updated.